### PR TITLE
LPS-97199 Refactor

### PR DESCRIPTION
**Pull request by darquesdev**

### CODE REVIEW

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/AsahSegmentsExperimentMapper.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Sarai Díaz
+ * @author David Arques
+ */
+@Component(immediate = true, service = AsahSegmentsExperimentMapper.class)
+public class AsahSegmentsExperimentMapper {
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/AsahSegmentsExperimentMapper.java]: SegmentsExperimentDTOMapper

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/AsahSegmentsExperimentMapper.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.segments.asah.connector.internal.processor;
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/AsahSegmentsExperimentMapper.java]: client.converter

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/LayoutURLHelper.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+		}
+		else {
+			Company company = _companyLocalService.getCompany(
+				layout.getCompanyId());
+
+			virtualHostname = company.getVirtualHostname();
+		}
+
+		boolean secure = StringUtil.equalsIgnoreCase(
+			Http.HTTPS, "web.server.protocol");
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/LayoutURLHelper.java]: Change to key PropsValues.WEB_SERVER_PROTOCOL

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/LayoutURLHelper.java [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Sarai Díaz
+ * @author David Arques
+ */
+@Component(immediate = true, service = LayoutURLHelper.class)
+public class LayoutURLHelper {
+
+	public String getLayoutFullURL(Layout layout) throws PortalException {
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/LayoutURLHelper.java]: Move method to ExperimentDTOConverter file

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/LayoutURLHelper.java (hunk 3 above)
Comment (darquesdev) [reply modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/processor/LayoutURLHelper.java]: it does not work, i can not see PropsValues class

**Comment by dgarciasarai** [comment]
ci:close

**Pull request closed without merging**